homeassistant/components/smarttub/controller.py

```diff
@@ -54,8 +54,7 @@ async def async_setup_entry(self, entry):
             )
         except LoginFailed:
             # credentials were changed or invalidated, we need new ones
-
-            return False
+            raise ConfigEntryAuthFailed
```

Comment:
```suggestion
            raise ConfigEntryAuthFailed from ex
```

Reply:
Doesn't that happen automagically when raising a different exception in an except block? I made it explicit anyway

Reply:
Ah, I see, pylint complains about it